ui/pages/signup/index.js

```diff
@@ -39,9 +39,7 @@ export default function Signup() {
       // redirect to the new org subdomain
       let created = await res.json()
 
-      window.location = `${window.location.protocol}//${
-        created?.organization?.domain
-      }.${window.location.host.replace('www.', '')}`
+      window.location = `${window.location.protocol}//${created?.organization?.domain}`
```

Comment:
Just confirming, this returns the domain name with the base URL correct?

Ex:
`acme-123.app.infrahq.com`

Reply:
it does. You have to set the base domain with `--hostname infrahq.com` or similar in the config